src/main.cpp

```diff
@@ -37,22 +36,13 @@
 extern "C" {
 #include <libavutil/log.h>
 #include <rs.h>
```

Comment:
I'm not sure if any other includes can be removed from here now.

Reply:
I think most of those includes can probably go away now

Reply:
I think I removed everything that could be.

I never included `#include <libavutil/log.h>` anywhere else, so not sure about that?

Reply:
Hmm, that should be for the `av_log_*` functions used below, but it must be getting pulled in via some indirect dependency. It's probably a good idea to leave it there anyway.